### ISSUE TITLE
`minify_html` plugin dependency inaccessible, breaks build

### ISSUE THREAD

**Issue by pixeldesu** [lumeland/lume#689]
### Version

2.4.1

### Platform

Windows / Linux (Netlify Build)

### What steps will reproduce the bug?

Building a website with the `minify_html` plugin on a system without dependency caching

### How often does it reproduce? Is there a required condition?

Currently, all the time.

### What is the expected behavior?

Builds/Package installs pass.

### What do you see instead?

Relevant section of the Netlify deployment error:
```
5:03:37 AM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
5:03:37 AM: error: Uncaught (in promise) TypeError: Import 'https://wilsonl.in/minify-html/deno/0.15.0/index.js' failed: error sending request for url (https://wilsonl.in/minify-html/deno/0.15.0/index.js): error trying to connect: dns error: failed to lookup address information: No address associated with hostname
5:03:37 AM:     at https://deno.land/x/lume@v2.4.1/deps/minify_html.ts:2:24
5:03:37 AM:     const mod = await import(url);
5:03:37 AM:                 ^
5:03:37 AM:     at async createSite (https://deno.land/x/lume@v2.4.1/cli/utils.ts:26:17)
5:03:37 AM:     at async buildSite (https://deno.land/x/lume@v2.4.1/cli/utils.ts:45:16)
5:03:37 AM: 
```

### Additional information

The last successful build has been 3 hours ago at the time of writing (2:29 GMT+1), the next build at 3:12 AM and each subsequent one failed, until I [removed the `minify_html` plugin, including the broken dependency from deno.lock](https://github.com/pixeldesu/pixelde.su/commit/57a60a7c423c22d8a586751caa698afd261c0de2).

**Comment by oscarotero** [comment]
Seems that the serve is down (my fault for trusting this server).
Fortunately, I can recover the code from my local cache. I'm going to publish it on deno.land/x

**Comment by oscarotero** [comment]
Fixed in Lume 2.4.2
